src/snowflake/cli/plugins/nativeapp/codegen/snowpark/extension_function_utils.py

```diff
@@ -131,5 +131,5 @@ def _enrich_entity(
         handler=entity["handler"],
     )
     entity["all_imports"] = _get_all_imports(
-        raw_imports=entity["raw_imports"], suffix_str=suffix_str
+        raw_imports=entity["raw_imports"] or [], suffix_str=suffix_str
```

Comment:
Acking preventing errors such as "NoneType is not iterable". Does that mean that in my sanitize/add_defaults function, it would be better to have the default values be an empty value (list, dict, str) instead of None ?
That way, you won't need to keep track of it when calling functions.

Reply:
I prefer to deal with a model that's been sanitize, personally. This was just a quick way to achieve this for processors in particular.

Reply:
Okay, I'll edit the defaults in my PR to not always be None.